quarkus-artemis-ra: tests fail with quarkus 3.30.0.CR1

When building with quarkus `3.30.0.CR1`, the tests for `quarkus-artemis-ra` fail with the following error:

<details>
<summary>Test error</summary>

See: https://github.com/quarkiverse/quarkus-artemis/actions/runs/19337960401/job/55318460881#step:4:617

```
java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
Error: ]: Build step io.quarkus.arc.deployment.ArcProcessor#generateResources threw an exception: java.lang.reflect.UndeclaredThrowableException
	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:878)
	at io.quarkus.builder.BuildContext.run(BuildContext.java:255)
	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2651)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2630)
	at org.jboss.threads.EnhancedQueueExecutor.runThreadBody(EnhancedQueueExecutor.java:1622)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1589)
	at java.base/java.lang.Thread.run(Thread.java:1583)
	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
Caused by: java.util.concurrent.ExecutionException: java.lang.ClassCastException: class io.quarkus.arc.processor.BeanGenerator$3 cannot be cast to class io.quarkus.gizmo.MethodCreator (io.quarkus.arc.processor.BeanGenerator$3 and io.quarkus.gizmo.MethodCreator are in unnamed module of loader io.quarkus.bootstrap.classloading.QuarkusClassLoader @7708b66a)
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122)
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191)
	at io.quarkus.arc.processor.BeanProcessor.generateResources(BeanProcessor.java:440)
	at io.quarkus.arc.deployment.ArcProcessor.generateResources(ArcProcessor.java:538)
	at java.base/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:733)
	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:872)
	... 8 more
Caused by: java.lang.ClassCastException: class io.quarkus.arc.processor.BeanGenerator$3 cannot be cast to class io.quarkus.gizmo.MethodCreator (io.quarkus.arc.processor.BeanGenerator$3 and io.quarkus.gizmo.MethodCreator are in unnamed module of loader io.quarkus.bootstrap.classloading.QuarkusClassLoader @7708b66a)
	at io.quarkus.arc.processor.BeanGenerator.lambda$generateDestroy$66(BeanGenerator.java:1644)
	at io.quarkus.gizmo2.impl.BlockCreatorImpl.accept(BlockCreatorImpl.java:990)
	at io.quarkus.gizmo2.impl.TryCreatorImpl.lambda$body$0(TryCreatorImpl.java:41)
	at io.quarkus.gizmo2.impl.BlockCreatorImpl.nesting(BlockCreatorImpl.java:1514)
	at io.quarkus.gizmo2.impl.TryCreatorImpl.body(TryCreatorImpl.java:40)
	at io.quarkus.arc.processor.BeanGenerator.lambda$generateDestroy$70(BeanGenerator.java:1484)
	at io.quarkus.gizmo2.impl.TryCreatorImpl.accept(TryCreatorImpl.java:130)
	at io.quarkus.gizmo2.impl.BlockCreatorImpl.try_(BlockCreatorImpl.java:1153)
	at io.quarkus.arc.processor.BeanGenerator.lambda$generateDestroy$71(BeanGenerator.java:1483)
	at io.quarkus.gizmo2.impl.BlockCreatorImpl.accept(BlockCreatorImpl.java:990)
	at io.quarkus.gizmo2.impl.ExecutableCreatorImpl.doCode(ExecutableCreatorImpl.java:388)
	at io.quarkus.gizmo2.impl.ExecutableCreatorImpl.lambda$doBody$1(ExecutableCreatorImpl.java:304)
	at io.github.dmlloyd.classfile.impl.DirectCodeBuilder.build(DirectCodeBuilder.java:93)
	at io.github.dmlloyd.classfile.impl.DirectMethodBuilder.withCode(DirectMethodBuilder.java:118)
	at java.base/java.util.ArrayList$ArrayListSpliterator.tryAdvance(ArrayList.java:1685)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:129)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:527)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:513)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:647)
	at org.junit.platform.engine.support.discovery.EngineDiscoveryRequestResolution.resolve(EngineDiscoveryRequestResolution.java:189)
	at org.junit.platform.engine.support.discovery.EngineDiscoveryRequestResolution.resolve(EngineDiscoveryRequestResolution.java:126)
	at org.junit.platform.engine.support.discovery.EngineDiscoveryRequestResolution.resolveCompletely(EngineDiscoveryRequestResolution.java:92)
	at org.junit.platform.engine.support.discovery.EngineDiscoveryRequestResolution.run(EngineDiscoveryRequestResolution.java:83)
	at org.junit.platform.engine.support.discovery.EngineDiscoveryRequestResolver.resolve(EngineDiscoveryRequestResolver.java:148)
	at org.junit.jupiter.engine.discovery.DiscoverySelectorResolver.resolveSelectors(DiscoverySelectorResolver.java:63)
	at org.junit.jupiter.engine.JupiterTestEngine.discover(JupiterTestEngine.java:70)
	at org.junit.platform.launcher.core.EngineDiscoveryOrchestrator.discoverEngineRoot(EngineDiscoveryOrchestrator.java:195)
	at org.junit.platform.launcher.core.EngineDiscoveryOrchestrator.discoverSafely(EngineDiscoveryOrchestrator.java:174)
Error:  TestEngine with ID 'junit-jupiter' encountered a critical issue during test discovery:
	at org.junit.platform.launcher.core.EngineDiscoveryOrchestrator.discover(EngineDiscoveryOrchestrator.java:119)

	at org.junit.platform.launcher.core.EngineDiscoveryOrchestrator.discover(EngineDiscoveryOrchestrator.java:84)
	at org.junit.platform.launcher.core.DefaultLauncher.discover(DefaultLauncher.java:104)
	at org.junit.platform.launcher.core.DefaultLauncher.discover(DefaultLauncher.java:83)
	at org.junit.platform.launcher.core.DelegatingLauncher.discover(DelegatingLauncher.java:42)
	at org.junit.platform.launcher.core.InterceptingLauncher.lambda$discover$0(InterceptingLauncher.java:33)
	at org.junit.platform.launcher.core.ClasspathAlignmentCheckingLauncherInterceptor.intercept(ClasspathAlignmentCheckingLauncherInterceptor.java:25)
	at org.junit.platform.launcher.core.InterceptingLauncher.discover(InterceptingLauncher.java:33)
	at org.junit.platform.launcher.core.DelegatingLauncher.discover(DelegatingLauncher.java:42)
	at org.apache.maven.surefire.junitplatform.LazyLauncher.discover(LazyLauncher.java:50)
	at org.apache.maven.surefire.junitplatform.TestPlanScannerFilter.accept(TestPlanScannerFilter.java:52)
	at org.apache.maven.surefire.api.util.DefaultScanResult.applyFilter(DefaultScanResult.java:87)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.scanClasspath(JUnitPlatformProvider.java:144)
	at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:124)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
Caused by: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
Error: ]: Build step io.quarkus.arc.deployment.ArcProcessor#generateResources threw an exception: java.lang.reflect.UndeclaredThrowableException
	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:878)
	at io.quarkus.builder.BuildContext.run(BuildContext.java:255)
	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2651)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2630)
	at org.jboss.threads.EnhancedQueueExecutor.runThreadBody(EnhancedQueueExecutor.java:1622)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1589)
	at java.base/java.lang.Thread.run(Thread.java:1583)
	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
Caused by: java.util.concurrent.ExecutionException: java.lang.ClassCastException: class io.quarkus.arc.processor.BeanGenerator$3 cannot be cast to class io.quarkus.gizmo.MethodCreator (io.quarkus.arc.processor.BeanGenerator$3 and io.quarkus.gizmo.MethodCreator are in unnamed module of loader io.quarkus.bootstrap.classloading.QuarkusClassLoader @7708b66a)
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122)
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191)
	at io.quarkus.arc.processor.BeanProcessor.generateResources(BeanProcessor.java:440)
	at io.quarkus.arc.deployment.ArcProcessor.generateResources(ArcProcessor.java:538)
	at java.base/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:733)
	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:872)
	... 8 more
```
</details>

--- a/pom.xml
+++ b/pom.xml
@@ -40,7 +40,7 @@
         <artemis.version>2.44.0</artemis.version>
         <camel-quarkus.platform.version>3.30.0.CR1</camel-quarkus.platform.version>
         <quarkus.version>3.30.0.CR1</quarkus.version>
-        <quarkus-ironjacamar.version>1.6.1</quarkus-ironjacamar.version>
+        <quarkus-ironjacamar.version>1.7.0.CR1</quarkus-ironjacamar.version>
 
         <!-- Project setup -->
         <maven.compiler.parameters>true</maven.compiler.parameters>

